Assets/Scripts/Models/Events/TimeManager.cs

```diff
@@ -120,13 +120,13 @@ public void Update(float time)
         InvokeEvent(EveryFrame, time);
 
         // Systems that update every frame not in Modal.
-        if (WorldController.Instance.IsModal == false)
+        if (GameController.Instance.IsModal == false)
         {
             InvokeEvent(EveryFrameNotModal, time);
         }
 
         // Systems that update every frame while unpaused.
-        if (WorldController.Instance.IsPaused == false)
+        if (IsPaused == false)
```

Comment:
Notice that ```IsPaused == false``` is not always the same as ```GameController.Instance.IsPaused == false```, since ```GameController.Instance.IsPaused``` also checks for ```GameController.Instance.IsModal```.